tools/release/release_utils.sh

```diff
@@ -66,6 +66,24 @@ function publishImages {
         echo "push image: $image"
         docker tag $image:$baseTag $image:$curTag
         retry 5 docker push $image:$curTag
+
+    done
+    docker logout
+}
+
+function publishPackages{
+    local curTag=$1
+    local baseTag=$2
+    local dockerHubUser=$3
+    local dockerHubPassword=$4
+    docker login ghcr.io -u $dockerHubUser -p $dockerHubPassword
```

Comment:
Login to ghcr.io cannot use docker credentials.

Reply:
please let me know what are the ghcr.io credentials.

Reply:
Use the credentials below while modifying images as parameters.
Modify the images of [publishImages](https://github.com/goharbor/harbor/blob/a79c0ad496e6f656c4150cc955e179848c935444/tools/release/release_utils.sh#L56) as parameters like this.
```    
local ghcrUser=$3
local ghcrPassword=$4
local images=$5
docker login ghcr.io -u $ghcrUser -p $ghcrPassword
```